general-concepts/slotting/text.xml

```diff
@@ -45,10 +52,108 @@ To <c>DEPEND</c> upon a package in a specific slot, refer to
 </p>
 </body>
 
+<section>
+<title>Considerations for slotting</title>
+<body>
+
+<p>
+Slotting is a powerful tool and as such, is easily abused. It's critical
+that when slotting, package maintainers carefully reflect on implications
+for <e>reverse dependencies</e>.
+</p>
+
+<p>
+Do not be fooled by the ease in adapting the package itself to slotting:
+slotting the package itself may well be the easiest part.
+</p>
+
+<p>
+Before proceeding at all, consider the wider implications beyond Gentoo.
+Especially examine other distributions and their handling of the problem in
+question. Unilaterally moving software can have devastating consequences for
+both users and other developers.
+</p>
+
+<p>
+Even if every possible consumer of a slotted package has been adapted in
+Gentoo, remember that people may develop or test software on Gentoo systems:
+they both want to be able to build that software without having to adapt
+to Gentooisms, and they also don't want to unintentionally write their
+software such that it relies on Gentooisms.
+</p>
+
+<p>
+Systems that lend themselves to slotting, such as Guile or Lua, cannot
+be naively slotted by just adapting the package itself. Other packages
+must be primed to handle a slotted landscape: an eclass mechanism usually
+needs to exist for packages to declare which versions they support, as well
+as force building against the version currently depended upon to avoid
+automagic dependencies.
+</p>
```

Comment:
I find these three paragraphs (but especially the last one) hard to understand. The text is somewhat vague and lacks specificity. Maybe it could profit from an example?

Reply:
Yes, that's better.